dev_requirements.txt

```diff
@@ -0,0 +1 @@
+zappa==0.50.0
```

Comment:
if this is required to deploy, then it's not a dev requirement?

Reply:
The scripts can be run directly from command line as well.